packages/components/pages/f-offers/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-offers",
   "description": "Fozzie Offers - displays offers to the customers",
-  "version": "1.12.0-beta.5",
+  "version": "1.13.0-beta.1",
```

Comment:
What was the reason to go to a new minor beta package, out of curiosity?

Reply:
Just consistency with the other pages to be honest!